src/sdk/namespace_fs.js

```diff
@@ -721,7 +721,7 @@ class NamespaceFS {
                         if (pos < results.length) {
                             results.splice(pos, 0, r);
                         } else {
-                            const stat = await native_fs_utils.stat_ignore_eacces(this.bucket_path, r, fs_context);
+                            const stat = await native_fs_utils.stat_ignore_eacces_enoent(this.bucket_path, r.key, fs_context);
```

Comment:
what will happen if file will get deleted after this check?

Reply:
I think it is related to the discussion [here](https://github.com/noobaa/noobaa-core/pull/8809#discussion_r1961276737), and I made some changes after thinking about it.